ThreeBotPackages/threefold/tfgrid_workloads/actors/workload_manager.py

```diff
@@ -146,21 +150,25 @@ def _reservation_check(self, jsxobj):
             else:
                 jsxobj.next_action = "invalid"
 
-        elif jsxobj.next_action == "sign":
+        if jsxobj.next_action == "sign":
             signatures = jsxobj.signatures_provision
             request = jsxobj.data_reservation.signing_request_provision
             if self._request_check(payload, request, signatures):
                 jsxobj.next_action = "pay"
 
-        elif jsxobj.next_action == "pay":
-            if self._validate_farmers_signature(jsxobj):
-                jsxobj.next_action = "deploy"
-
-        elif jsxobj.next_action == "deploy":
-            signatures = jsxobj.signatures_delete
-            request = jsxobj.data_reservation.signing_request_delete
-            if self._request_check(payload, request, signatures):
-                jsxobj.next_action = "delete"
+        if jsxobj.next_action == "pay":
+            # Temporary change to simplify reservation flow for testing
+            # if self._validate_farmers_signature(jsxobj):
+            jsxobj.next_action = "deploy"
+            pass
+
+        if jsxobj.next_action == "deploy":
```

Comment:
With is change we now cannot ask to delete a reservation anymore.
We should still have this possibility. But I agree we should also simplify this for now.
Maybe just adapt the `sign_delete` method to set the next_action to `delete` without checking signature ?

Reply:
can we also add a test case where we send the reservation with the customer signature already in and validate that the `next_action` value is `deploy`